src/lwc/alertBanner/alertBanner.js

```diff
@@ -0,0 +1,82 @@
+import { LightningElement, api } from 'lwc';
+import { isEmpty } from 'c/utilCommon';
+
+/**
+ * @typedef {Class} AlertBanner
+ * @property {string} _alertClass The class theme.
+ */
+export default class AlertBanner extends LightningElement {
+    /**
+     * The stylized theme for the alert.
+     * @type {string}
+     * @default warning
+     */
+    @api theme;
+    @api variant;
+    @api assistText;
+    @api icon;
+    @api size;
+    @api message;
+
+    defaultClass = 'slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_';
```

Comment:
Can this `defaultClass` field be made a constant of the module up above instead?